google/cloud/storage/bucket.py

```diff
@@ -84,6 +84,16 @@
 )
 _API_ACCESS_ENDPOINT = "https://storage.googleapis.com"
 
+PUBLIC_ACCESS_PREVENTION_ENFORCED = "enforced"
+"""Enforced public access prevention value.
+See: https://cloud.google.com/storage/docs/public-access-prevention
```

Comment:
That URL 404s -- is the backend feature not yet public?

Reply:
It is not yet public.